File: CODE FILES/Mobile/MyApiary/app/src/main/java/projects/myapiary/models/SensorsUpdateRequest.kt
package projects.myapiary.models

import com.google.gson.annotations.SerializedName

data class SensorsUpdateRequest(
    @SerializedName("sensor_id")
    val id: Int,
    @SerializedName("minValue")
    val minvalue: Float,
    @SerializedName("maxValue")
    val maxvalue: Float,
    @SerializedName("isWorking")
    val isworking: Boolean?,
    @SerializedName("beehiveId")
    val beehiveId: Int
)

File: CODE FILES/Backend/MyApiary/Resource.Api/Models/SensorsUpdateRequest.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace Resource.Api.Models
{
    public class SensorsUpdateRequest
    {
        [Required]
        public int sensor_id { get; set; }

        public float minValue { get; set; }

        public float maxValue { get; set; }

        public bool isWorking { get; set; }

        [Required]
        public int beehiveId { get; set; }
    }
}
